### ISSUE TITLE
feature: allow choosing custom build profiles

### ISSUE THREAD

**Issue by jtmoon79** [flamegraph-rs/flamegraph#257]
tl;dr allow passing `--profile` to `cargo flamegraph`

### Problem

In `Cargo.toml`, I optimized the release profile for the binary.
```lang-toml
[profile.release]
codegen-units = 1
panic = "abort"
strip = true
```

Now `cargo flamegraph` fails with
```lang-text
$ cargo flamegraph --bin my_program
...
Error: unable to generate a flamegraph from the collapsed stack data

Caused by:
    0: I/O error: No stack counts found
    1: No stack counts found
```

### Solution

Allow passing `--profile=profile_name` to `cargo flamegraph`, similar to `cargo build --profile=...`. Then `flamegraph` could then pass the `profile_name` to the underlying `cargo build --profile=` call. This would allow creating a custom profile for flamegraph in `Cargo.toml`.

For example, I would then be able to add to `Cargo.toml`:
```lang-toml
[profile.flamegraph]
inherits = "release"
codegen-units = 16
panic = "unwind"
strip = false
```

and then I could select that build profile with
```lang-text
$ cargo flamegraph --bin my_program --profile flamegraph
```

**Comment by djc** [comment]
Makes sense, want to submit a PR?

**Comment by jtmoon79** [comment]
> Makes sense, want to submit a PR?

I'll put it on my TODO list.

**Comment by jtmoon79** [comment]
@djc I think this feature was already implemented by @madadam in c532c046b171952f8812dd76b09298c0c558bf8d (Issue #167 )

https://github.com/flamegraph-rs/flamegraph/blob/8792e0927c43a53c80584948622ce7be259e60e6/src/bin/cargo-flamegraph.rs#L15-L17
```rust
    /// Build with the specified profile
    #[clap(long)]
    profile: Option<String>,
```

Could a new version of flamegraph be released?

**Comment by djc** [comment]
Released https://github.com/flamegraph-rs/flamegraph/releases/tag/v0.6.3 last week.
